libff/algebra/curves/tests/test_groups.cpp

```diff
@@ -317,6 +317,26 @@ void test_bls12_377()
     ASSERT_EQ(bls12_377_G2::zero(), z);
 }
 
+// chack that some elements e.g. 1,-1,2,random satisfy the curve
+// equation Y^2 = X^3 + a X + b; used in test_bls12_381
+template<typename GroupT> void check_curve_equation(GroupT P)
+{
+    P.to_affine_coordinates();
+    using Fq = typename std::decay<decltype(P.X)>::type;
+    Fq LHS = (P.Y * P.Y);
+    Fq RHS = ((P.X * P.X * P.X) + (GroupT::coeff_a * P.X) + GroupT::coeff_b);
+    ASSERT_EQ(LHS, RHS);
```

Comment:
Let's not use capitalized names here (this is generally used for constants). Instead let's use `lhs` and `rhs`

Reply:
Done